pom.xml

```diff
@@ -51,6 +51,9 @@
 
     <spotbugs.effort>Max</spotbugs.effort>
     <spotbugs.threshold>Low</spotbugs.threshold>
+    <!--TODO Until is included in parent pom -->
+    <maven.compiler.release>17</maven.compiler.release>
+    <jenkins-test-harness.version>2250.v03a_1295b_0a_30</jenkins-test-harness.version>
```

Comment:
This change is not needed because of the good work that was done elsewhere in the pull request to replace the use of Jetty as an HTTP server with the HTTP server that is available with Java.
```suggestion
```